map/map-test.js

```diff
@@ -1044,3 +1042,39 @@ QUnit.test('Deep updating a map', function() {
 	QUnit.equal(obj.list[0], 'something', 'the first element of the list should be updated');
 
 });
+
+testHelpers.dev.devOnlyTest("Error on not using a constructor or string on short-hand definitions (#278)", function() {
+	expect(1);
+	var message = "A Constructor or string must be used with shorthand definitions: https://canjs.com/doc/can-define.types.propDefinition.html#String";
+	var finishErrorCheck = testHelpers.dev.willError(message);
+
+	// Having JSHint ignore this because we don't need to actually
+	// use 'VM' anywhere to get the errors to show and the test to complete
```

Comment:
Just calling `DefineMap.extend({ })` without setting it to a variable would work.